### PR TITLE
Learn Site changes and add new course under a learning path

### LINKED ISSUE #1940: [Learn][Epic] Add a course to the learn-ng site
**Current State:**
The current learn section doesn't have an actual course present within the learning path.

**Desired State:**
A course needs to be added, named `Introduction to Service Mesh - Hands On`. We can start with migrating the labs in the `istio-service-mesh-workshop` course at https://github.com/layer5io/istio-service-mesh-workshop as individual chapters for this course.

This issue can be considered as an epic and, a PR for each chapter/lab can be taken up for migration.

Figma Design for the sample page can be found [here](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=5820%3A12409). The sidebar component is not included in this issue.

---
**Contributor Resources**
The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- Check the Learn Layer5 site [Design Spec](https://docs.google.com/document/d/1tQ9tLbetBVKP7z76FR19vsxbAfoFHZzZYxCt7lJpAhU/edit) to know more about the site structure and development plans.
- Check out the Learn Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/?node-id=4853%3A0).

### PR DESCRIPTION
**Pull request by sudo-NithishKarthik**
Signed-off-by: suod-NithishKarthik <nithishkarthik01@gmail.com>

Changes to the learn-ng site and add a new course under 'mastering-meshery-istio' named '
Introduction to Service Mesh - Hands On'

This PR fixes  #1940, #1866

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit cc402df6b4cf240754ae3cf0e44d189494177c50 at: https://60d5f0f56a81fa547c6a736f--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 18a98cf472f8436462de48ff34de623460231272 at: https://60d5f3f744093558e6f16698--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 99b7a49a2a74b5d38e1c2668a8e5505522ed5625 at: https://60d61074c17c0e7043ac26f4--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit add3e8e13a83902594f542ab8b8835354a91c089 at: https://60d6139cfe405f7cf68957ff--layer5ng.netlify.app

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: src/components/Learn-Components/Course-Overview-page/index.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ const CourseOverview = ({ course, chapters }) => {
             {chapters.map((chapter, index) => (
               <Link
                 key={index}
-                to={`./istio/${chapter.fields.chapter}`}
+                to={`/learn-ng${chapter.fields.slug}`}
```
[the remark this answers is not in the document]
Comment (sudo-NithishKarthik) [reply src/components/Learn-Components/Course-Overview-page/index.js:58]: Yes, this makes a lot of sense now !

File: content-learn/mastering-meshery-istio/introduction-to-service-meshes/index.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
```
[the remark this answers is not in the document]
Comment (sudo-NithishKarthik) [reply content-learn/mastering-meshery-istio/introduction-to-service-meshes/index.mdx:1]: Sure

File: content-learn/mastering-meshery-istio/introduction-to-service-meshes/chapter-1/section-1.mdx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+docType: "Chapter"
+chapterTitle: "Chapter One"
+description: "Meshery is the service mesh managemen plane which offers lifecycle, configuration, and performance management of service meshes and their workloads."
+videos: 4
+lectures: 12
+---
+
+Hello World
```
[the remark this answers is not in the document]
Comment (sudo-NithishKarthik) [reply content-learn/mastering-meshery-istio/introduction-to-service-meshes/chapter-1/section-1.mdx:9]: Sure

File: src/components/Learn-Components/LearnInfo/index.js [hunk 4]
```diff
@@ -1,30 +1,38 @@
...
+  }
+  courses:allMdx(
+    filter: {
+      fields: {collection: {eq: "content-learn"}},
+      frontmatter: { docType: {eq: "Course"}}
+    }
+  ) {
+    totalCount
+  }}
+  `;
```
[the remark this answers is not in the document]
Comment (sudo-NithishKarthik) [reply src/components/Learn-Components/LearnInfo/index.js:26]: Ok

**Pull request closed without merging**